tracetools/propagate.go

```diff
@@ -3,7 +3,7 @@ package tracetools
 import (
```

Comment:
It'll also be interesting to see if we can use the proposed env vars in https://github.com/open-telemetry/opentelemetry-specification/issues/740 (which is also mentioned in https://github.com/buildkite/agent/issues/1663) as a move towards more standardization. This will help others reusing libraries native to whatever language/tool they're using while still maintaining trace linking across process boundaries.